### DISCUSSION TITLE
Try a test meeting

### DISCUSSION THREAD

**Discussion by ackernaut** [ParabolInc/parabol#6659] [Ideas]
From [this user feedback](https://www.loom.com/share/d958ddc72d4a4dfa928a06244069e9b2) (~17:00) 🔒 they stated there wasn’t really a way to ‘test a meeting’.

Along the lines of “don’t make me think”, not everybody thinks to create test teams in the moment. What if we did it for them?

![try-a-test-meeting](https://user-images.githubusercontent.com/307286/171444413-5a012df0-a709-4b68-90bd-07d496140c99.png)

### Idea
- Somewhere on the start meeting view there’s an action to ‘try a test meeting’
- Clicking this will check if a team ‘My Test Team’ exists. If the user has this team, it will use it. If they don’t the team will be created for the user
- The meeting type is opened on the test team instead of the selected team. The functionality is the same. They could invite some colleagues to kick the tires if they wanted etc.
- When sharing usage related to ‘active teams’ we may need heuristic to filter out test teams. We probably already do that today to an extent in usage reports to customers

**Note**: I used [the new dialog view](https://www.figma.com/file/5ouU5EQOZ0ELcBA0xWZGdQ?node-id=1407:11827#204453613) (Figma), but we could try it on the current view

**Comment by acressall** [comment]
Ooh thanks for putting this idea together @ackernaut! I think we should either implement the new meeting view first, or try this on the current view so we get clear results about which aspect is having impact. 
A few questions:
- Would the CTA go away if the user has done a test meeting before? ie. I clicked it the first time around, now I'm going for my second meeting
- Should there also be a CTA to start "for real" or something so it's clear that you can still do that from here?

**Reply by ackernaut** [reply]
>Would the CTA go away if the user has done a test meeting before?

I suppose this could be noisy after a while. I wonder if it would be better suited as a card on the meeting view. There’s 1 ‘try a test Meeting Type’ per meeting type that goes away if they try it or click X to remove. I think we could try something similar and cheap though, where it’s just a real meeting on a test team

>Should there also be a CTA to start "for real" or something

I wouldn’t change the standard behavior or label. The ‘test’ CTA is the outlier

Just ideas! Those are helpful questions for testing the idea!!

**Reply by ackernaut** [reply]
What I think I like the most is making it easy to start a real meeting on a test team without them having to think about it. If we like the idea we could try it in different ways:
- As a special Meetings View card
- As a secondary action on the start view
- Somewhere else?

**Comment by enriquesanchez** [comment]
I _**love**_ the idea of allowing users to try a test meeting, but I don't think the new meeting dialog is the right place for it. We have to be mindful of how much we ask from users here. If we add too many options and choices to a UI where users already have to pick a meeting type, a team, a name, a template and the icebreaker, it quickly becomes overwhelming. I also think that for the sake of clarity, no other action should compete with `Start meeting`.

We can look at how other tools do onboarding, and take some inspiration from them. For example, I wonder if giving all new users some "pre-opened" meetings that by default appear in their meetings view (instead of an empty state). Like how Trello gives you a new board with some content preloaded. Or Notion starts you off with a document that has some initial content.

We can have one for each meeting type, and make it clear with text explanations that these are "sandbox" meetings. Users can play in them with no risks, these don't generate a timeline event, or send out email summaries. 

These sandbox meetings should have some default content added to them, to avoid throwing users into empty meetings with no clear direction. We can have some reflections, comments, tasks, etc. that make the sandbox meeting experience richer, and help users understand what they can do with that particular meeting type. Basically a meeting tutorial inside the meeting.

**Reply by ackernaut** [reply]
>I also think that for the sake of clarity, no other action should compete with Start meeting.

I like this and agree with the thinking (I love sharing bad ideas)! As soon as I posted this I thought it would probably be better as a meeting card that said try a test [meeting type].

I like the idea of sandbox meetings. I could see how the script to pre-load content could be tricky, it’s basically what we’re doing for the demo (except there’s the complexity of timing actions of the bots). I’d be okay with empty meetings without content since it’s a cheap thing to try. Their first meeting of each type on the team is opened automatically. I’m not sure what we’d measure but it could make for a fun experiment. I’d vote on it 😁
